### ISSUE TITLE
Getting Not Allowed error when connecting to Amazon Cloud Drive from DSM 6.1

### ISSUE THREAD

**Issue by esbenvb** [duplicati/duplicati#2412]
I have:
- [x] searched open and closed issues for duplicates

----------------------------------------

### Version info

**Duplicati Version:** 2.0.1.53
**Operating System:** Synology DSM 6.1-15047 Update 2
**Backend:** Amazon Cloud Drive

### Bug description
Cannot connect to Amazon Cloud drive. I get an error every time I try. Works fine in same Duplicati verison on my Mac, same amazon account. 

**EDIT: I ran the command /var/packages/Mono/target/usr/local/bin/cert-sync /etc/ssl/certs/ca-certificates.crt and now the certificate-error in the log is not shown any more. But I still get the error: Not Allowed when I try to log in. And there's nothing in the log files.**

### Steps to reproduce
- Installed Mono Beta 4.6.2-0094 package (the stable package wouldn't even start Duplicati) 
- Create backup
- Storage type: Amazon Cloud Drive
- Click AuthId, authorized amazon cloud
- Enter a folder name, click test

**Actual result:** I get a "Failed to connect:" error.

**Expected result:** Expected to get access to my files on Amazon Cloud Drive
### Screenshots
<img width="1920" alt="screen shot 2017-03-29 at 21 57 05" src="https://cloud.githubusercontent.com/assets/426051/24474018/939ead48-14cb-11e7-8bcd-abce74f3e2bd.png">
<img width="336" alt="screen shot 2017-03-30 at 00 34 07" src="https://cloud.githubusercontent.com/assets/426051/24479694/bfe9e952-14e0-11e7-980d-01c33490abf8.png">
### debug log

29. mar 2017 22:04: Request for http://diskstation.local:5000/api/v1/remoteoperation/test gave error
Duplicati.Library.Utility.SslCertificateValidator+InvalidCertificateException: The server certificate had the error RemoteCertificateChainErrors and the hash 1CCBD698BEA48E80826C03379E78468CD3582C71
If you trust this certificate, use the commandline option --accept-specified-ssl-hash=1CCBD698BEA48E80826C03379E78468CD3582C71 to accept the server certificate anyway.
You can also attempt to import the server certificate into your operating systems trust pool.
You may want to import a set of trusted certificates into the Mono certificate store.
Use the command:
    cert-sync /etc/ssl/certs/ca-certificates.crt #for Debian based systems
    cert-sync /etc/pki/tls/certs/ca-bundle.crt #for RedHat derivatives
Read more: http://www.mono-project.com/docs/about-mono/releases/3.12.0/#cert-sync
  at Duplicati.Library.Utility.SslCertificateValidator.Deactivate () [0x00048] in <df389ec2993a4c5d84cc50d023650ee9>:0 
  at Duplicati.Library.Utility.SslCertificateValidator.Dispose () [0x0000b] in <df389ec2993a4c5d84cc50d023650ee9>:0 
  at Duplicati.Library.Modules.Builtin.HttpOptions.Dispose () [0x0003e] in <aa6ffe4eb9c34c81a1cb8f93ea411ac4>:0 
  at Duplicati.Server.WebServer.RESTMethods.RemoteOperation.TestConnection (System.String url, Duplicati.Server.WebServer.RESTMethods.RequestInfo info) [0x001c3] in <ea04d3246d8b40a8b87fd599cff89cc3>:0 
  at Duplicati.Server.WebServer.RESTMethods.RemoteOperation.POST (System.String key, Duplicati.Server.WebServer.RESTMethods.RequestInfo info) [0x000af] in <ea04d3246d8b40a8b87fd599cff89cc3>:0 
  at Duplicati.Server.WebServer.RESTHandler.DoProcess (Duplicati.Server.WebServer.RESTMethods.RequestInfo info, System.String method, System.String module, System.String key) [0x002ad] in <ea04d3246d8b40a8b87fd599cff89cc3>:0

**Comment by kenkendk** [comment]
Can you try this:
https://github.com/duplicati/duplicati/wiki/SSL-TLS-support-in-Mono#synology

**Comment by Thenollyon** [comment]
After some experimenting, I was able to resolve the certificate issues I was experiencing on my Synology by performing a cert-sync as sudo [https://github.com/duplicati/duplicati/issues/2431#issuecomment-296371556](url) 

I also remember getting a Not Allowed message when the folder did not exist on Amazon Cloud Drive. Perhaps my notes are off but I remember that I then went out to Amazon and created a new folder before entering that new folder name into Duplicati. I no longer get the Not Allowed message.

Hope this helps someone else.
